.pre-commit-config.yaml

```diff
@@ -9,6 +9,7 @@ repos:
       - id: codespell
         additional_dependencies:
           - tomli
+        exclude: utils/airport_timezones.json
```

Comment:
Was this necessary for you? I have this configuration in `pyproject.toml` so codespell should exclude it without needing this. I would prefer to have this in only one place, not both as it will be easier to add/remove configurations.

Reply:
It was triggering for me if i run `pre-commit run --all-files`, but I'm happy to remove it if you don't want it there.
<img width="646" alt="image" src="https://github.com/jdholtz/auto-southwest-check-in/assets/5343931/f972114a-27f5-426a-aede-9a2f06251e8d">

Reply:
Ah, yes I get the same issue. Seems to be a bug with codespell then.